docs-source/content/faq/namespace-management.md

```diff
@@ -0,0 +1,168 @@
+
+```
+
+#### Recreating a previously deleted Kubernetes namespace
+
+If you need to delete a namespace (and the resources in it) and then recreate it,
+remember to remove the namespace from the operator's `domainNamespaces` list
+after you delete the namespace, and add it back to the `domainNamespaces` list after you recreate the namespace
+using the `helm upgrade` commands that were illustrated previously.
+
+{{% notice note %}}
+Make sure that you wait a sufficient period of time between deleting and recreating the
+namespace because it takes time for the resources in a namespace to go away after the namespace is deleted.
+In addition, as mentioned above, changes to the `domainNamespaces` setting is monitored by the operator
+periodically, and the operator becomes ready to host domain resources only after the required domain
+`configmap` (namely `weblogic-domain-cm`) is initialized in the namespace.
+{{% /notice %}}
+
+If a domain custom resource is created before the namespace is ready, you might see that the introspector job pod
+fails to start, with a warning like the following, when you review the description of the introspector pod.
+Note that `domain1` is the name of the domain in the following example output.
+
+```
+Events:
+  Type     Reason                 Age               From               Message
+  ----     ------                 ----              ----               -------
+  Normal   Scheduled              1m                default-scheduler  Successfully assigned domain1-introspect-domain-job-bz6rw to slc16ffk
+
+  Normal   SuccessfulMountVolume  1m                kubelet, slc16ffk  MountVolume.SetUp succeeded for volume "weblogic-credentials-volume"
+
+  Normal   SuccessfulMountVolume  1m                kubelet, slc16ffk  MountVolume.SetUp succeeded for volume "default-token-jzblm"
+
+  Warning  FailedMount            27s (x8 over 1m)  kubelet, slc16ffk  MountVolume.SetUp failed for volume "weblogic-domain-cm-volume" : configmaps "weblogic-domain-cm" not found
+
+```
+
+If you run into situations where a new domain that is created after its namespace is deleted and recreated,
+does not come up successfully, you can restart the operator pod as shown in the following examples, where
+the operator itself is running in the namespace `weblogic-operator-namespace` with a release name
+of `weblogic-operator`.
+
+* Kill the operator pod, and let Kubernetes restart it.
+
+```
+$ kubectl delete pod/weblogic-operator-65b95bc5b5-jw4hh -n weblogic-operator-namespace
+```
+
+* Scale the operator deployment to `0` and then back to `1` by changing the value of the `replicas`.
+
+```
+$ kubectl scale deployment.apps/weblogic-operator -n weblogic-operator-namespace --replicas=0
+```
+
+```
+$ kubectl scale deployment.apps/weblogic-operator -n weblogic-operator-namespace --replicas=1
+```
+
+{{% notice note %}}
+Restarting an operator pod interrupts not just the domain resources in the namespace that is to be re-initialized,
```

Comment:
What does interrupt mean here?  Does the operator introspect the domain and do a restart of the domain pods?

Reply:
No, the existing domain pods will not be interrupted. The only interruption is the management of the domains via the operator, for example, creating a new domain in any of the namespaces that the operator manages. But the interruption is only  for a short period of time; when the operator is being restarted.  I'll change the wording to be clearer. I am also thinking of changing the "notice note" to be a regular note so it is less scary.

Reply:
Updated the last section. Thanks @pfmackin.